src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -87,8 +87,12 @@ namespace ngsiv2
   /* ****************************************************************************
   *
   * Notification::toJson -
+  *
+  * FIXME P2: we should move 'attrsFormat' from Subject class to Notification
+  * class, to avoid passing attrsFormat as argument
+  *
```

Comment:
In order not to introduce noise in the template and blacklist functionality, that may be using affected parts in Subscriptions classes and corresponding parsing and rendering logic.

Thus, it is better to do it after template and blacklist functionality land into develop.